### PR TITLE
The nightly che-plugin-registry job fails on the "InstallPluginUsingUI" test

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Rework test for not using API

### What issues does this PR fix or reference?

Issue: https://github.com/eclipse/che/issues/19871

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/e2e/tests/plugins/InstallPluginUsingUI.spec.ts [hunk 1]
```diff
@@ -9,43 +9,43 @@
...
 const projectTree: ProjectTree = e2eContainer.get(CLASSES.ProjectTree);
 const pluginsView: PluginsView = e2eContainer.get(CLASSES.PluginsView);
 
-const testWorkspaceUtils: TestWorkspaceUtil = e2eContainer.get<TestWorkspaceUtil>(TYPES.WorkspaceUtil);
 const browserTabsUtil: BrowserTabsUtil = e2eContainer.get(CLASSES.BrowserTabsUtil);
 const workspaceHandling: WorkspaceHandlingTests = e2eContainer.get(CLASSES.WorkspaceHandlingTests);
 
-const workspaceName: string = 'install-plugin-test';
-const workspaceUrl: string = `${TestConstants.TS_SELENIUM_BASE_URL}/dashboard/#/ide/${TestConstants.TS_SELENIUM_USERNAME}/${workspaceName}`;
+const devfileUrl: string = 'https://raw.githubusercontent.com/eclipse/che/master/tests/e2e/files/devfiles/plugins/InstallPluginUsingUI.yaml';
```
Comment (dmytro-ndp) [inline tests/e2e/tests/plugins/InstallPluginUsingUI.spec.ts]: `master` branch has been renamed to `main`

Comment (Ohrimenko1988) [reply tests/e2e/tests/plugins/InstallPluginUsingUI.spec.ts]: Link works in both cases (master and main). Anyway fix implemented.

**Comment by Ohrimenko1988** [comment]
[crw-ci-test]

**Review by dmytro-ndp — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
[crw-ci-test]

**Pull request merged**